src/items/appItemHooks.ts

```diff
@@ -129,7 +123,15 @@ const useAppCommunication = ({
         iFrameRef?.current?.contentWindow?.postMessage(
           JSON.stringify({
             type: POST_MESSAGE_KEYS.GET_CONTEXT_SUCCESS,
-            payload: contextPayload,
+
+            payload: {
+              /**
+               * @deprecated use accountId.
```

Comment:
Not sure we need to mark this as deprecated ? But I think it is important to keep the comment that explains why this is here.